cmake/configure/configure_1_cuda.cmake

```diff
@@ -19,26 +19,29 @@
 
 MACRO(FEATURE_CUDA_FIND_EXTERNAL var)
 
+  IF(${DEAL_II_WITH_CUDA})
 
-  FIND_PACKAGE(CUDA)
+    FIND_PACKAGE(CUDA)
 
-  IF(CUDA_FOUND)
+    IF(CUDA_FOUND)
 
-    IF(NOT DEAL_II_WITH_CXX11)
-      MESSAGE(FATAL_ERROR "\n"
-        "CUDA only supported with C++11. Reconfigure with DEAL_II_WITH_CXX11=ON.\n"
-        )
-    ENDIF()
+      IF(NOT DEAL_II_WITH_CXX11)
+        MESSAGE(FATAL_ERROR "\n"
+          "CUDA only supported with C++11. Reconfigure with DEAL_II_WITH_CXX11=ON.\n"
+          )
+      ENDIF()
```

Comment:
As a general guideline. Please, don't put a FATAL_ERROR message into the configuration stage :-)